src/main/java/org/jabref/model/strings/StringUtil.java

```diff
@@ -755,4 +755,12 @@ public static String removeStringAtTheEnd(String string, String stringToBeRemove
     public static boolean endsWithIgnoreCase(String string, String suffix) {
         return StringUtils.endsWithIgnoreCase(string, suffix);
     }
+
+    /**
+     * @param string input string
+     * @return  {@link Optional#empty()} if the {@param string} is empty, otherwise wrap it in {@link Optional}.
+     */
+    public static Optional<String> optionalOfEmpty(String string) {
+        return Optional.ofNullable(string).filter(s -> !s.isEmpty());
```

Comment:
there is a similar existing method already isNotBlank which takes already an optional